src/test/java/io/mathdojo/QuestionFunctionTest.java

```diff
@@ -0,0 +1,19 @@
+package io.mathdojo;
+
+public class QuestionFunctionTest {
```

Comment:
We need to modify this so that we run some unit tests.

Reply:
I modified them and will add more